tests/osautoinst/test_running.pm

```diff
@@ -4,7 +4,7 @@ use testapi;
 
 sub run {
     assert_script_run 'command -v ack >/dev/null || zypper --no-refresh -n in ack';
-    assert_script_run 'ret=false; for i in {1..5} ; do openqa-client jobs state=running | ack --passthru --color running && ret=true && break ; sleep 30 ; done ; [ "$ret" = "true" ]', 300;
+    assert_script_run 'ret=false; for i in {1..5} ; do openqa-client jobs state=running | ack --passthru --color 'running|done' && ret=true && break ; sleep 30 ; done ; [ "$ret" = "true" ]', 300;
```

Comment:
if this works, does that mean that `openqa-client jobs state=running` does not really filter for "running" and instead simply shows all jobs?

Reply:
oops. , how embarassing.

I now added `state=done` and also switched to the new client.